std_msgs/Header header
bool brake_status
float32 vbus_voltage
float32 ibus
float32 motor_temperature
float32 input_position
float32 input_velocity
float32 input_torque
float32 motor_torque
float32 motor_pos
float32 motor_vel
float32 sync_roller_pos
float32 sync_roller_vel
int32 sync_roller_raw
